debugger/src/main/resources/messages/RsDebuggerBundle.properties

```diff
@@ -0,0 +1,15 @@
+rust.debugger.renderers.bundled.item=Bundled renderers
+rust.debugger.renderers.compiler.item=Rust compiler''s renderers
```

Comment:
Seems like `'` shouldn't be rendered twice here

<img src="https://user-images.githubusercontent.com/6342851/151161953-ffaf6bfe-7642-4862-853b-06073048eaca.png" width="500">

Reply:
Nice catch 👍 
Thanks!